### PR TITLE
Misc fix ups to match updates to WIlls PR 

### PR DESCRIPTION
**Pull request by fluffy**

### CODE REVIEW

**Review by suhasHere — APPROVED** [review]

File: draft-lcurley-moq-transport.md [hunk 1]
```diff
@@ -495,16 +501,16 @@ OBJECT message header carry short hop-by-hop Track Id that maps to the Full Trac
 
 ## Relay Discovery and Failover
 
-TODO: This section shall cover aspects of relay failover and protocol interactions
+TODO: This section shall cover aspects of relay failover and protocol interactions.
```
Comment (afrind) [inline draft-lcurley-moq-transport.md:504]: I thought I saw this `.` appear in a different PR too, hopefully no merge conflicts :P

File: draft-lcurley-moq-transport.md [hunk 2]
```diff
@@ -213,24 +213,25 @@ It is up to the application building on top of MoQ to define how broad or narrow
 An application that deals with connections between devices on a local network may limit the scope to a single connection;
 by contrast, an application that uses multiple CDNs to serve media may require the scope to include all of those CDNs.
 
-The full track name is the only piece of information that is used to identify the track within a given MoQ scope and is used as key for caching.
+The full track name is the only piece of information that is used to
+identify the track within a given MoQ scope and that  is used as a key for caching.
```
Comment (afrind) [inline draft-lcurley-moq-transport.md:217]: minor minor nit: two spaces between `that  is`

**Review by afrind — COMMENTED** [review]
LGTM

File: draft-lcurley-moq-transport.md [hunk 3]
```diff
@@ -283,7 +284,10 @@ Full Track Name = security-camera.example.com/camera1/hd-video
 
 ### Connection URL
 
-Each track MAY have one or more associated connection URLs specifying network hosts through which a track may be accessed. The syntax of the Connection URL and the associated connection setup procedures are specific to the underlying transport protocol usage {{transport-protocols}}.
+Each track MAY have one or more associated connection URLs specifying
```
Comment (kixelated) [inline draft-lcurley-moq-transport.md:287]: I'm confused by the old text. Tracks don't "have connection URLs".

**Pull request merged**